### DISCUSSION TITLE
5.0.0-beta.10 文档不完善

### DISCUSSION THREAD

**Discussion by yaderick** [antvis/G2#4973] [Q&A]
- node.encode() 参数类型
- node.style() 有哪些参数
- 目前 interval 有以下几个内置 shape 图形，默认为 rect。怎么绘制 hollow | 绘制空心的矩形

**Comment by pearmini** [comment]
- node.encode 的参数可以看这里的教程：https://g2.antv.antgroup.com/manual/single-view/mark#%E9%80%9A%E9%81%93

![image](https://user-images.githubusercontent.com/49330279/219592851-3d529842-bc6d-415f-9bae-035b1e583ff1.png)

- node.style 和 interval 的内置图形可以看这里：https://g2.antv.antgroup.com/api/mark/interval#%E9%80%89%E9%A1%B9

文档会在正式版本的时候更加详细。

**Comment by coader** [comment]
I  can search 'legend' in doc site, but after click it, the result page can't found any words about 'legend'

**Comment by pearmini** [comment]
> legend

@hustcc 看一下这个和官网搜索的问题？我试了一下确实不行。

**Comment by hustcc** [comment]
@yaderick 搜索已经 ok 了，文档我们也会在近两个月好好整理一下，特别是 API 文档。
